### ISSUE TITLE
Implement Redux Actions for Medical Copays

### ISSUE THREAD

**Issue by LinnJS** [department-of-veterans-affairs/va.gov-team#27601]
## Technical Task 
Setup Redux actions to fetch mock user copay data

- [VBS.Api Swagger Docs](https://yf7yvyucfj.execute-api.us-gov-west-1.amazonaws.com/Prod/swagger/index.html)
- [Mocks](https://preview.uxpin.com/18073b171729031ac0af64822d31a7427b012d66#/pages/139873795/simulate/sitemap?mode=i)

## Expected Behavior
Redux actions are configured to fetch copay data and app container is receiving mock data

## Acceptance Criteria
- [ ] _Add bulleted list of all the task that bust be finished for the feature to be consider finished & ready to merge_

**Comment by LinnJS** [comment]
@craigwheeler and I moved this from a 8 to a 5 due to less unknown with the BE integration

**Comment by LindaLi123** [comment]
Blocked by endpoint data unavailable

**Comment by craigwheeler** [comment]
should be able to set up redux actions using mock data to unblock FE work for now and will create a new ticket to follow up when endpoint data is available
